policyengine_canada/parameters/gov/provinces/nu/tax/income/credits/cost_of_living/rate.yaml

```
@@ -0,0 +1,9 @@
+description: Nunavut cost of living credit rate.
```

Could you phrase all parameter descriptions like this?
```suggestion
description: Nunavut phases in its cost of living credit at this rate with respect to income.
```